cvat/apps/cf/README.md

```diff
@@ -0,0 +1,12 @@
+## Git Integration For Annotation Storage
+
+### Description
+
+The application allows you to integrate CVAT task metrics with CloudFactory's Workstreams Pulse offering to measure and improve task performance for an individual CloudWorker or team of CloudWorkers
```

Comment:
Could you please give me an idea that it is? Just give me a screenshot or link on documentation. Just want to understand how it will help other users or how to use it internally and why it is better than our analytics component.

Reply:
@nmanovic I prematurely included the CF app I've been building and this readme was associated with it. It's gone now. Apologies!

I'll provide full documentation and screenshots for the CF app integration when I get that ready for a PR.  Essentially, we have our own workforce platform that we use to manage Cloud Workers and sending task completion data to it helps us better manage the teams doing the work for clients using CloudFactory.  More on that when I complete the app.